packages/documentation/src/pages/platform/index.mdx

```diff
@@ -1,5 +1,32 @@
 ---
 title: Platform docs
+tags: platform, architecture, front end standards, tools, unit tests
 ---
 
-Here's where documentation about the platform lives.
+## About the Platform
```

Comment:
I think "Platform" can be lower case here. Our headings are typically sentence case